docs/Lama2/docs/explanation/l2format.md

```diff
@@ -83,6 +83,17 @@ Cookies are specified in a `Cookie` header as follows:
 Cookie:'sessionid=foo;another-cookie=bar'
 ```
 
+### Global Environments variables/commands can be defined in `/l2config.env/<requests_dir>`
```

Comment:
Instead of calling it "global", I think we should call it something like:

1. Project-level variables
2. Root variables
3. Project-root variables

Let's pick one of the above and finalize (or make other suggestions).

Second, I think we should use `git` repo analogy to explain.

Reply:
I have added the git repo analogy in example.md, should I link it here as well?

Reply:
Yes